include/config/config_debug.h

```diff
@@ -94,3 +94,9 @@
  * Intentionally crash the game whenever a runtime assertion fails (also invoked by the DEBUG define in the Makefile).
  */
 #define DEBUG_ASSERTIONS
+
+/**
+ * Crash Screen: enable symbol-assisted stack trace. Adds a bit more code to (somewhat) intelligently walk the stack.
+ *  If disabled, stack trace will just dump the memory at the stack pointer, from which one can cross-reference with the function map.
+ */
+#define DEBUG_FULL_STACK_TRACE
```

Comment:
why is this a define

Reply:
Code size has inflated a lot compared to the old implementation that just walked the stack, enough that a release build of a given project might want to shed it off

Reply:
kind of pointless if disable all doesn't catch it. if people are expected to know enough to know it exists it should be in a different file anyway

Reply:
Addressed by moving to a makefile switch